nengo/simulator.py

```diff
@@ -134,6 +134,7 @@ def __init__(
             dt=0.001, seed=None, model=None, progress_bar=True, optimize=True):
         self.closed = True  # Start closed in case constructor raises exception
         self.progress_bar = progress_bar
+        self.optimize = optimize
```

Comment:
Update docstring attributes to include `optimize` (and also `progress_bar` which was missing) to be consistent with how other things like `nengo.[Ensemble|Connection|etc...]` list any attributes that were passed in through `__init__`.